web_src/js/bootstrap.js

```diff
@@ -20,6 +20,11 @@ export function showGlobalErrorMessage(msg) {
  * @param {ErrorEvent} e
  */
 function processWindowErrorEvent(e) {
+  if (!e.error && e.lineno === 0 && e.colno === 0 && e.filename === '' && window.navigator.userAgent.includes('FxiOS/')) {
```

Comment:
Isn't the last condition unnecessary?
It feels less like a workaround without the last condition...
```suggestion
  if (!e.error && e.lineno === 0 && e.colno === 0 && e.filename === '') {
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox#firefox_for_ios

[Firefox for iOS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox#firefox_for_ios)

Firefox for iOS uses the default Mobile Safari UA string, with an additional FxiOS/<version> token on iPod and iPhone

Reply:
At the moment.
But if Firefox already manages to produce such an obscure bug, why shouldn't the others be able to do that as well?

Reply:
I do not think it could be a general bug for modern browsers. This bug is highly likely to be caused some incorrect DOM layout handling. The last similar bug in my mind is as old as back to Internet Explorer 6 (20 years ago) ......